model/clusters_mgmt/v1/upgrade_policy_state_type.model

```diff
@@ -16,8 +16,9 @@ limitations under the License.
 
 // Representation of an upgrade policy state that that is set for a cluster.
 class UpgradePolicyState {
-	// State value, can be 'pending', 'started', 'delayed', 'failed' or 'completed'.
-	Value String
+	// State value can be 'pending', 'scheduled', 'canceled', 'started', 'delayed', 
```

Comment:
In the enum above you have 'cancelled', but here it says 'canceled' - I think we should use the 'cancelled' spelling variant, as it applies more internationally